NickvisionTagger.Shared/Models/MusicFile.cs

```diff
@@ -85,9 +85,10 @@ static MusicFile()
         {
             _invalidSystemFilenameCharacters.Remove('\\');
         }
-        else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+        else
         {
             _invalidSystemFilenameCharacters.Remove('/');
+            _invalidWindowsFilenameCharacters.Remove('/');
```

Comment:
Well, `_invalidWindowsFilenameCharacters` is only used on non-Windows platforms for the Limit Filename Characters preference anyway, so maybe just not include `/` :sweat_smile:

Reply:
`_invalidSystemFilenameCharacters.Remove(Path.DirectorySeparatorChar);`